lightmd/render/html_render.py

```diff
@@ -9,46 +10,96 @@ def get_html_format(node):
+        return f"<code{style_class}>{{}}</code>"
     if isinstance(node, HeaderParagraph):
-        return "<h{level}>{{}}</h{level}>".format(level=node.level())
+        level = node.level()
+        style_class = get_style_class(f"h{level}")
+        return f"<h{level}{style_class}>{{}}</h{level}>"
 
     _format = ""
     if isinstance(node, TextParagraph):
-        _format = "<p>{}</p>"
+        style_class = get_style_class("p")
+        _format = f"<p{style_class}>{{}}</p>"
     if isinstance(node, ListWrapper):
         if node.is_ordered():
-            _format = "<ol>{}</ol>"
+            style_class = get_style_class("ol")
+            _format = f"<ol{style_class}>{{}}</ol>"
         else:
-            _format = "<ul>{}</ul>"
+            style_class = get_style_class("ul")
+            _format = f"<ul{style_class}>{{}}</ul>"
     if isinstance(node, QuoteParagraph):
-        _format = "<blockquote>{}</blockquote>"
+        style_class = get_style_class("blockquote")
+        _format = f"<blockquote{style_class}>{{}}</blockquote>"
 
     # Block
     if isinstance(node, TextBlock):
-        return '{}'
+        return "{}"
     if isinstance(node, ImgBlock):
-        return '<img src="{0}" alt="{1}">'.format(node.url(), node.content())
+        style_class = get_style_class("img")
+        return f'<img src="{node.url()}" alt="{node.content()}"{style_class}>'
     if isinstance(node, LinkBlock):
-        return '<a href="{0}">{1}</a>'.format(node.url(), node.content())
+        style_class = get_style_class("href")
+        return f'<a href="{node.url()}"{style_class}>{node.content()}</a>'
     if isinstance(node, CodeBlock):
-        return "<code>{}</code>"
+        style_class = get_style_class("code")
+        return f"<code{style_class}>{{}}</code>"
 
     if isinstance(node, ListParagraph):
-        _format = "<li>{}</li>"
+        style_class = get_style_class("li")
+        _format = f"<li{style_class}>{{}}</li>"
     if isinstance(node, BoldBlock):
-        _format = "<strong>{}</strong>"
+        style_class = get_style_class("strong")
+        _format = f"<strong{style_class}>{{}}</strong>"
     if isinstance(node, ItalicBlock):
-        _format = "<em>{}</em>"
+        style_class = get_style_class("em")
+        _format = f"<em{style_class}>{{}}</em>"
     if isinstance(node, StrikethroughBlock):
-        _format = "<del>{}</del>"
+        style_class = get_style_class("del")
+        _format = f"<del{style_class}>{{}}</del>"
     if isinstance(node, QuoteParagraph):
-        _format = "<blockquote>{}</blockquote>"
+        style_class = get_style_class("blockquote")
+        _format = f"<blockquote{style_class}>{{}}</blockquote>"
 
     return _format
 
-def render(root):
-    #print(HTML_FORMAT[HorizontalRule])
+
+def render(root, css_path=None):
+    # print(HTML_FORMAT[HorizontalRule])
+    global STYLE_MAP
+    STYLE_MAP = parse_css(css_path)
     return root.render(get_html_format)
+
+
+def parse_css(css_path):
```

Comment:
We don't require that method.

Reply:
Based on our last discussion, I decide to stay with this approach. It allows users to specify multiple classes for one kind of element.